CantoboardFramework/Keyboard/View/LayoutConstants.swift

```diff
@@ -343,9 +343,9 @@ class LayoutConstants: Copyable {
     var numOfSingleCharCandidateInRow: Int {
         switch idiom {
         case .phone:
-            return isPortrait ? 8 : 15
+            return isPortrait ? 10 : 18
```

Comment:
Could we keep this unchanged? A lot of users are complaining the candidates are too small already